config/settings/development.yml

```diff
@@ -43,3 +43,5 @@ autotest:
 lti:
   domains: <%= %w[host.docker.internal] %>
   token_endpoint: "http://host.docker.internal:3100/login/oauth2/token"
+
+web_socket_allowed_request_origins: <%= %w[host.docker.internal] %>
```

Comment:
Let's nest this configuration option under `rails` -> `action_cable`, so that it better matches the structure of existing options.